### ISSUE TITLE
Electroencephalogram based Control of Prosthetic Hand using Optimizable Support Vector Machine

### ISSUE THREAD

**Issue by eyouso** [eyouso/CapstoneResources#8]
### Title
Electroencephalogram based Control of Prosthetic Hand using Optimizable Support Vector Machine
### URL
https://dl.acm.org/doi/10.1145/3610419.3610453
### Summary 
How SVMs were used to work with prosthetic hand control
### Key Points 
Bayesian optimizer, accuracy, usage of SVMs
### Citation
@inproceedings{10.1145/3610419.3610453,
author = {Pooya Chanu, Maibam and Pei, Dingyi and Olikkal, Parthan and Vinjamuri, Ramana Kumar and Kakoty, Nayan M},
title = {Electroencephalogram based Control of Prosthetic Hand using Optimizable Support Vector Machine},
year = {2023},
isbn = {9781450399807},
publisher = {Association for Computing Machinery},
address = {New York, NY, USA},
url = {https://doi.org/10.1145/3610419.3610453},
doi = {10.1145/3610419.3610453},
abstract = {Research on electromyogram (EMG) controlled prosthetic hands has advanced significantly, enriching the social and professional lives of people with hand amputation. Even so, the non-functionality of motor neurons in the remnant muscles impedes the generation of EMG as a control signal. However, such people have the same ability as healthy individuals to generate motor cortical activity. The work presented in this paper investigates electroencephalogram (EEG)-based control of a prosthetic hand. EEG of 10 healthy subjects performing the grasping operations were acquired for classification of hand movements. 15 EEG channels were selected to classify hand open and close operations. Hand movement-class-specific time-domain features were extracted from the filtered EEG. A support vector machine (SVM) was employed with 24-fold cross-validation for classification using extracted features. SVM hyper-parameters for the classification model were optimized with a Bayesian optimizer with a minimum prediction error as an objective function. During training and testing of the classifier model, an average accuracy of 96.8 ± 0.98\% and 93.4 ± 1.16\% respectively, were achieved across the subjects. The trained classifier model was employed to control prosthetic hand open and close operations. This study demonstrates that EEG can be used to control a prosthetic hand by amputees with motor neuron disabilities.},
booktitle = {Proceedings of the 2023 6th International Conference on Advances in Robotics},
articleno = {34},
numpages = {6},
keywords = {Electroencephalogram, Prosthetic hand, Support Vector Machine},
location = {, Ropar, India, },
series = {AIR '23}
}
